web3.js/test/connection.test.ts

```diff
@@ -186,71 +186,59 @@ describe('Connection', () => {
   });
 
   it('get multiple accounts info', async () => {
-    const account1 = Keypair.generate();
-    const account2 = Keypair.generate();
 
-    {
-      await helpers.airdrop({
-        connection,
-        address: account1.publicKey,
-        amount: LAMPORTS_PER_SOL,
-      });
+    const multipleAccounts = [];
+
+    for (let i = 0; i < 100; i++) {
+      multipleAccounts.push(Keypair.generate());
+    }
 
+    for (const account of multipleAccounts) {
       await helpers.airdrop({
         connection,
-        address: account2.publicKey,
+        address: account.publicKey,
         amount: LAMPORTS_PER_SOL,
       });
```

Comment:
Each airdrop has to wait for the last to finish because `await` is being called in a loop, which is a big deal when there are hundreds of accounts! See if you can figure out how to use `Promise.all` to await them _all_ in parallel.